drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/emergency_shelter_allowance_grant_loader.rb

```diff
@@ -0,0 +1,73 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# matriculation to new platform
+module HmisExternalApis::AcHmis::Importers::Loaders
+  class EmergencyShelterAllowanceGrantLoader < CustomDataElementBaseLoader
+    def filename
+      'EmergencyShelterAllowanceGrant.csv'
+    end
+
+    protected
+
+    def cde_definitions_keys
+      [
+        :esg_allowance_grant_referred,
+        :esg_allowance_grant_received,
+        :esg_allowance_grant_received_amount,
+        :esg_allowance_grant_reason_not_referred,
+      ]
+    end
+
+    def build_records
```

Comment:
Maybe I'm missing something obvious, but where does this method get called? Other loaders have a `perform`

Reply:
So this should work as follows:
* This class inherits from CustomDataElementBaseLoader which defines perform
* CustomDataElementBaseLoader.perform calls build_records

It felt convenient and DRY to use inheritance here but it is a little indirect and hard to follow; lmk if I should refactor

Reply:
Oh okay gotcha, thanks!